packages/next-codemod/transforms/__testfixtures__/next-async-request-api-dynamic-props/access-props-28.output.tsx

```diff
@@ -0,0 +1,5 @@
+export { /* Next.js Dynamic Async API Codemod: `default` export is re-exported. Check if this component uses `params` or `searchParams`*/
```

Comment:
This should read 
```suggestion
export { /* Next.js Dynamic Async API Codemod: `Page` export is re-exported. Check if `Page` uses `params` or `searchParams`*/
```

you can use the local identifier instead of the re-exported one. It's not `default` that's re-exported but `Page`.